tutor4points/core/templates/users_profile.html

```diff
@@ -18,130 +18,180 @@
-                                    </div>
-                                </div>
-
-                                {{form_update_profile.is_tutor | as_crispy_field}}
-
-                                <div class = "form-row" id = "tutor-form">
-                                    <div class="col-6">
-                                        {{form_update_profile.classes_taken | as_crispy_field}}
-                                    </div>
-                                    <div class="col-6">
-                                        {{form_update_profile.times_available | as_crispy_field}}
-                                    </div>
-                                    <div class="col-6">
-                                        {{form_update_profile.time_zone | as_crispy_field}}
-                                    </div>
-                                    <div class="col-6">
-                                        {{form_update_profile.rate | as_crispy_field}}
-                                    </div>
-                                </div>
-                              
-                                <button type="submit" class="btn-block btn-secondary">Update</button>
-                              
-                            </form>
+                        {{form_update_profile.is_tutor | as_crispy_field}}
+
+                        <div class="form-row" id="tutor-form">
+                            <div class="col-6">
+                                {{form_update_profile.classes_taken | as_crispy_field}}
+                            </div>
+                            <div class="col-6">
+                                {{form_update_profile.times_available | as_crispy_field}}
+                            </div>
+                            <div class="col-6">
+                                {{form_update_profile.time_zone | as_crispy_field}}
+                            </div>
+                            <div class="col-6">
+                                {{form_update_profile.rate | as_crispy_field}}
+                            </div>
                         </div>
-                    </div>
+
+                        <button type="submit" class="btn-block btn-secondary">Update</button>
+
+                    </form>
                 </div>
             </div>
-        </div>   
+        </div>
+    </div>
+</div>
 
-{% if form_update_profile.errors %}
+{% if current_user and form_update_profile.errors %}
 <script>
-    $(document).ready (function(){
+    $(document).ready(function () {
         $("#edit-profile-modal").modal('show')
     })
 </script>
 {% endif %}
 
+<!-- insert request form here -->
+<div class="modal fade" id="tutor-request-modal">
+    <div class="modal-dialog model-lg">
+        <div class="modal-content">
+            <div class="modal-flex-container">
+                <div class="form-container">
+                    <h2>Request</h2>
```

Comment:
The Request is centered on the one on the tutors.html page, but not for this one. It'll be best if the modal styles are exactly the same for both pages